action_files/test_models/src/models2.py

```diff
@@ -2,35 +2,39 @@
 import time
 
 import fire
-import numpy as np
+# import numpy as np
 import pandas as pd
-import pytorch_lightning as pl
-import torch
+# import pytorch_lightning as pl
+# import torch
 
-import neuralforecast
+# import neuralforecast
 from neuralforecast.core import NeuralForecast
 
 from neuralforecast.models.gru import GRU
-from neuralforecast.models.rnn import RNN
-from neuralforecast.models.tcn import TCN
+# from neuralforecast.models.rnn import RNN
+# from neuralforecast.models.tcn import TCN
 from neuralforecast.models.lstm import LSTM
 from neuralforecast.models.dilated_rnn import DilatedRNN
-from neuralforecast.models.deepar import DeepAR
-from neuralforecast.models.mlp import MLP
-from neuralforecast.models.nhits import NHITS
-from neuralforecast.models.nbeats import NBEATS
+# from neuralforecast.models.deepar import DeepAR
+# from neuralforecast.models.mlp import MLP
+# from neuralforecast.models.nhits import NHITS
+# from neuralforecast.models.nbeats import NBEATS
```

Comment:
I'm kind of treating this file also as a testing file locally, we can delete it (it's mainly so that testing locally is faster that you don't need to type all that stuff every time)